lib/BUILD

```diff
@@ -2,6 +2,14 @@ load("@bazel_skylib//:bzl_library.bzl", "bzl_library")
 
 licenses(["notice"])
 
+objc_library(
+    name = "swizzle_absolute_xcttestsourcelocation",
+    testonly = True,
+    srcs = ["swizzle_absolute_xcttestsourcelocation.m"],
+    visibility = ["//visibility:public"],
+    alwayslink = True,
+)
+
```

Comment:
How should we document this?

We need to mention how `BUILD_WORKSPACE_DIRECTORY` needs to be set. Also its main use will be through the `apple.swizzle_absolute_xcttestsourcelocation` `feature` that `rules_apple` and rules_swift` will honor and automatically add this dependency to the test targets.

Reply:
(And by how, I also mean literally. I don't think it will get picked up by Stardoc, so at which location as well.)

Reply:
If this is only going to be referenced internally in the other rules maybe just document in those test rules?

Reply:
That's the main use case, so I'm fine with that. Technically it's public API, so I'll add something right above this target for adventurous folks.